statistics/cmsketch.go

```diff
@@ -329,7 +325,12 @@ func (c *CMSketch) queryHashValue(h1, h2 uint64) uint64 {
 	return uint64(res)
 }
 
-func (c *CMSketch) mergeTopN(lTopN map[uint64][]*TopNMeta, rTopN map[uint64][]*TopNMeta, numTop uint32, usingMax bool) {
+// MergeTopN ...
+func MergeTopN(dst, src *TopN, c *CMSketch, numTop uint32, usingMax bool) {
+	if dst.TotalCount() == 0 || src.TotalCount() == 0 {
+		return
+	}
+	lTopN, rTopN := dst.topN, src.topN
```

Comment:
Would it be a little confusing to use `dst.topN, src.topN` and `lTopN, rTopN` interchangeably in this function? I think we can get rid of `lTopN` and `rTopN`.